simulation/__main__.py

```diff
@@ -65,12 +68,28 @@ def restrict_between_0_and_1(input_value: float):
         help="A factor by which current (2021) air source heat pump unit+install costs will have declined by, as of the end of 2022",
     )
 
+    def is_isoformat_datetime_string(string):
```

Comment:
Function name hints at a boolean being returned, but its a string. I believe you are using the `fromisoformat()` method to catch any errors in string formatting. Perhaps the following is easier to read:
```suggestion
    def check_string_is_isoformat_datetime(string) -> str:
```